src/participant_network.star

```diff
@@ -80,100 +78,85 @@ def launch_participant_network(
     final_genesis_timestamp = get_final_genesis_timestamp(
         plan, CL_GENESIS_DATA_GENERATION_TIME + num_participants * CL_NODE_STARTUP_TIME
     )
-    plan.print("Generating EL data")
-    el_genesis_generation_config_template = read_file(
-        static_files.EL_GENESIS_GENERATION_CONFIG_TEMPLATE_FILEPATH
-    )
-    el_genesis_data = el_genesis_data_generator.generate_el_genesis_data(
-        plan,
-        el_genesis_generation_config_template,
-        final_genesis_timestamp,
-        network_params.network_id,
-        network_params.deposit_contract_address,
-        network_params.genesis_delay,
-        network_params.seconds_per_slot,
-        network_params.capella_fork_epoch,
-        network_params.deneb_fork_epoch,
-        network_params.electra_fork_epoch,
-    )
 
-    plan.print(json.indent(json.encode(el_genesis_data)))
+    total_number_of_validator_keys = 0
+    for participant in participants:
+        total_number_of_validator_keys += participant.validator_count
 
-    plan.print("Uploading GETH prefunded keys")
+    plan.print("Generating EL CL data")
+    # we are running capella (deprecated)
```

Comment:
we are running bellatrix in the first case right?

Reply:
Yes, but its no longer supported.